src/app/components/event-page/event-page.component.html

```diff
@@ -0,0 +1,84 @@
+<div id="event-card"> 
+    <div class="flex-row" id="top-div">
+        <img id="event-image" [src]="transformStringToImage() ? transformStringToImage() : ''" alt="">
```

Comment:
set the result of this function on a property on the event model. 
ex event.ImageSrc